packages/gitbook/src/routes/ogimage.tsx

```diff
@@ -89,12 +89,15 @@ export async function serveOGImage(baseContext: GitBookSiteContext, params: Page
             await Promise.all(
                 primaryFontWeights.map((face) => {
                     const { weight, sources } = face;
-                    if (sources.length === 0) {
+                    const source = sources[0];
```

Comment:
Why are we taking the first source like that? Shouldn't we iterate? What are the sources?

Reply:
I' not sure tbh, I'm just adapting the code not to crash. Maybe @emmerich knows?